app/services/badge_granter.rb

```diff
@@ -46,6 +46,7 @@ def grant
     return if @granted_by && !Guardian.new(@granted_by).can_grant_badges?(@user)
     return unless @badge.present? && @badge.enabled?
     return if @badge.badge_grouping_id == BadgeGrouping::GettingStarted && @badge.id != Badge::NewUserOfTheMonth && @user.user_option.skip_new_user_tips
+    return unless @user.present?
```

Comment:
Given the line before refers to `@user`, shouldn't this check precede that?

Also: `if @user.blank?` 🏌️⛳ 😉

Reply:
Indeed that is better, thanks @CvX